src/pages/api/v1/discussions/add-reply-post.ts

```diff
@@ -0,0 +1,79 @@
+import prisma from "@/lib/prisma";
+import { NextApiResponse, NextApiRequest } from "next";
+import { withMethods } from "@/lib/api-middlewares/with-method";
+import { withAuthentication } from "@/lib/api-middlewares/with-authentication";
+import { errorHandler } from "@/lib/api-middlewares/errorHandler";
+import { getToken } from "next-auth/jwt";
+import { getCookieName } from "@/lib/utils";
+
+/**
```

Comment:
The API description should be `Post reply on a query`

Reply:
resolved